The automake option `dist-lzma' is obsoleted by `dist-xz'.

Signed-off-by: Alexandru-Cezar Sardan <alexandru.sardan@freescale.com>
---

--- a/Watchdog/configure.ac
+++ b/Watchdog/configure.ac
@@ -40,7 +40,7 @@
 AC_CANONICAL_HOST
 
 # Use Automake.
-AM_INIT_AUTOMAKE([foreign no-define dist-bzip2 dist-zip dist-lzma silent-rules 1.11])
+AM_INIT_AUTOMAKE([foreign no-define dist-bzip2 dist-zip dist-xz silent-rules 1.11])
 AC_SUBST(VERSION)
 
 # Generate a configuration header file.
